week-1/K-functions-parameters/exercise2.js

```diff
@@ -1,9 +1,11 @@
 // Declare your function first
-
+function divide() {
+  return 3 / 4;
+}
 var result = divide(3, 4);
 
```

Comment:
Same comments apply here as in exercise K-1, make use of function parameters to return a dynamic result.